### ISSUE TITLE
Extension to Classification

### ISSUE THREAD

**Issue by JanoschMenke** [YannDubs/Neural-Process-Family#12]
How would one want to expand the NPs, lets say the LNP, to the case classification.
I cant simply predict the location and scale of the Normal of the output distribution. I assume you need to make use of the logits.
